### ISSUE TITLE
Erkling and hair dye

### ISSUE THREAD

**Issue by archmancer8** [Kitteh6660/Corruption-of-Champions-Mod#188]
At max level, with full speed, a character can never be caught by the Erkling, even at maxed fatigue. It'd be kind of nice to still have access to the scene with his hounds without having to take a hit to pc speed.
Also, horse tails retain fur color, even after dyeing pc's hair a different color (i.e. have brown fur, dye hair blonde, tail is still described as brown).

**Comment by Darth-Cthoras** [comment]
If you look at Corruption-of-Champions-Mod/classes/classes/Scenes/Areas/Forest/ErlKingScene.as, you'll see there's quite a few modifiers to lower your score ... and to raise it - speed isn't the only factor, but certain body parts, as well, modify the hunt score. Did you check over the math yet to discern if maxed level, maxed speed, could still not be caught at full fatigue (120, as far as I know, with perks), if they had enough negative body modifiers?

**Comment by archmancer8** [comment]
Sheesh, I've got most of those positive attributes and none of the negatives. Max fatigue with ascension and perks can max at 170 actually. Also, max stats can fluctuate between species; the character I tested on is a horsemorph with a speed of 215 and intelligence of 190. 
I also found this in the scenes:

public function playerHuntScore():int
		{
			trace("Calculating Wild Hunt score.");
			trace("Int + Spd = " + String(player.inte + player.spe));
			trace("Base = " + String((player.inte + player.spe) - (player.fatigue * 2)));
			var baseVal:int = (player.inte + player.spe) - (player.fatigue * 2);

So, far as I can tell, my character's Wild Hunt score without modifiers and max fatigue should be 65, and 135 with modifiers. Which makes sense. I can get caught by the Erkling, as my score is over 100 but below 150 at max (170) fatigue. Like you said, I can make the clean get away most days, as my Hunt Score is over 150. The problem is it's impossible to get my score below 100, which is what flags the hound gangbang. Any thoughts, or should we hope for Kitteh?
